<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sift</class>
 <widget class="QMainWindow" name="sift">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>193</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>466</width>
    <height>193</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>sift</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad1">
        <property name="text">
         <string>Load from file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCamera1">
        <property name="text">
         <string>Capture from camera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnShow1">
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnROI">
        <property name="text">
         <string>Select Object</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnShowObject">
        <property name="text">
         <string>Show object</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Image 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad2">
        <property name="text">
         <string>Load from file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCamera2">
        <property name="text">
         <string>Capture from camera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnShow2">
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>SIFT</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDetectSIFT">
        <property name="text">
         <string>Detect SIFT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDetectObject">
        <property name="text">
         <string>Detect object</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbFetures">
        <property name="text">
         <string>Fetures</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbLines">
        <property name="text">
         <string>Lines</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbBox">
        <property name="text">
         <string>Box</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
